packages/instant/src/components/ui/input.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+
+import { ColorOption, styled } from '../../style/theme';
+
+export interface InputProps {
+    className?: string;
+    value?: string;
+    width?: string;
+    fontSize?: string;
+    fontColor?: ColorOption;
+    placeholder?: string;
+    onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+const PlainInput: React.StatelessComponent<InputProps> = ({ value, className, placeholder, onChange }) => (
+    <input className={className} value={value} onChange={onChange} placeholder={placeholder} />
+);
+
+export const Input = styled(PlainInput)`
+    font-size: ${props => props.fontSize};
+    width: ${props => props.width};
+    padding: 0.1em 0em;
+    font-family: 'Inter UI';
```

Comment:
Would it make sense for this font to be part of the theme?

Reply:
I think not for now... we actually only have 1 font in the entire widget I think

Reply:
Yea for now I'm thinking that the `theme` represent the configurable aspect of our UI... and for now that's just the colors... but open to changing that. The input prop could also just take a `fontFamily` prop -- but we can add that when we need it :D